sources/packages/web/src/helpers/helpers/uri.ts

```diff
@@ -71,16 +71,22 @@ export const getPaginationQueryString = (
   );
   // Sort parameters.
   if (paginationOptions.sortField && paginationOptions.sortOrder) {
-    const sortDBOrder =
-      paginationOptions.sortOrder === DataTableSortOrder.DESC ||
-      DataTableSortByOrder.DESC
-        ? FieldSortOrder.DESC
-        : FieldSortOrder.ASC;
+    let sortDBOrder = "";
```

Comment:
what is the difference here?

Reply:
The logic was wrong. Try passing `paginationOptions.sortOrder` with value "asc" for example.